src/main/java/com/github/monster860/fastdmm/editing/SelectPlacementMode.java

```diff
@@ -14,7 +14,7 @@
 public class SelectPlacementMode implements PlacementMode {
 	public FastDMM editor;
 	
-	public Set<Location> selection = new HashSet<Location>();
+	public static Set<Location> selection = new HashSet<Location>();
```

Comment:
no. no. no. no. no. Please don't do this, if you really need the contents of this elsewhere grab the `SelectPlacementMode` somewhere and access it trough it's reference, or put this elsewhere.